environment/docker-compose.yml

```diff
@@ -44,6 +44,9 @@ services:
     ports:
       - 8080:8080
 
+  mysql-cli:
```

Comment:
@sweiyang , why need this additional mysql-cli?

Reply:
i'll remove it